### ISSUE TITLE
v3 Split Table not working for me on staging sight on Chrome

### ISSUE THREAD

**Issue by Gazook89** [naturalcrit/homebrewery#1477]
v3/Chrome 92/macOS

On the staging site, with the Split Table snippet, I do not see the second table before adding any styling or anything else.  I can see that it is "there", but just not visible.   On Firefox I do not have this issue.

<img width="802" alt="image" src="https://user-images.githubusercontent.com/58999374/127406039-b7a69fa0-0587-4dc1-862c-4b2e01eb08d8.png">

I've had a few false alarms lately so I'm happy to be shown what I'm doing wrong.   [And here is a Share link demonstrating this](https://homebrewery-stage.herokuapp.com/share/72uDphw_TzNC)   It also includes alternate styling that uses Flex again, which i've also copied below:

```
.splitTable {
	display:flex;
	flex-wrap:wrap;
	border:1px dashed;
	border-radius:5px
}

.splitTable table {
	flex:1;
}

.splitTable :is(h1,h2,h3,h4,h5) {
	flex:1 0 100%;
}
```

**Comment by ericscheid** [comment]
The legacy split table does work. The only difference between the legacy split table `div` and the v3 split table `div` is 

- `break-inside: avoid;`
- `-webkit-transform: translateZ(0);`

Disabling the latter makes the default split table work.

**Comment by calculuschild** [comment]
Yep that pesky translate. It was the only solution to some other weirdness a while back with shadows/borders being cut off at column boundaries but we had to remove it from Legacy because it was messing with nested elements getting oddly positioned or something. I probably left it in v3 by mistake.

Not sure if the broken shadows ever got solved by a Chrome update but yep... Better to have broken shadows than disappearing items.
